src/patternfly/components/List/examples/List.md

```diff
@@ -87,3 +158,5 @@ Non-inline lists can be nested up to any level.
 | `.pf-c-list` | `<ul>, <ol>` | Initiates a list. **Required**  |
 | `.pf-m-inline` | `.pf-c-list` | Displays list items inline. |
 | `.pf-m-plain` | `.pf-c-list` |  Removes the list marker and base indentation. |
+| `.pf-m-bordered` | `pf-c-list` | Add horizontal divider between items in a list. |
```

Comment:
It might be a little confusing that we call it a horizontal divider but don't use the divider component. This is a little simpler. 

```suggestion
| `.pf-m-bordered` | `pf-c-list` | Adds a border between list items. |
```